packages/liveblocks-yjs/src/doc.ts

```diff
@@ -0,0 +1,106 @@
+import { detectDupes } from "@liveblocks/core";
+import { Base64 } from "js-base64";
+import { Observable } from "lib0/observable";
+import * as Y from "yjs";
+
+import { PKG_FORMAT, PKG_NAME, PKG_VERSION } from "./version";
+
+detectDupes(PKG_NAME, PKG_VERSION, PKG_FORMAT);
```

Comment:
this only needs to be on index.ts, right @nvie ?

Reply:
My understanding is that it should only be in `index.ts`

Reply:
Correct! It must be loaded exactly once, at the root of the module.